pages/linux/gsettings.md

```diff
@@ -0,0 +1,32 @@
+# gsettings
+
+> query and modify dconf settings with schema validation
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_the_desktop_environment_in_rhel_8/configuring-gnome-at-low-level_using-the-desktop-environment-in-rhel-8#using-gsettings-command_configuring-gnome-at-low-level>.
```

Comment:
This link is way to long I feel.

And from what I see, maybe we should use the manned.org link

Reply:
It is unfortunately very long, but the Redhat documentation does contain examples and links to learn more about the workings of dconf and gsettings.  Also, I feel that anyone who can `tldr gsettings` can just as well `man gsettings` on their own machine.  And may in fact have just *come* from the manpage.

Reply:
I'll look for an official site rather than a guide, if you can find one though since you know more about the topic, that would be better

Reply:
It's unfortunate that it's so long, but it seems quite good and far more usable than the official GNOME documentation or the man page.

Reply:
One option might be the developer documentation for GSettings itself, which is part of GLib.  The url is not overlong: https://developer.gnome.org/gio/stable/settings.html.  It doesn't really talk about the command line tool `gsettings` at all, but AFAICT, `gsettings` is a fairly thin wrapper around a few of the functions there, and if you do a deep-dive on it you'll probably get a better understanding than the man page.

Reply:
I don't think that page would be helpful for anyone wanting to use the command, but the `redhat` page still links to the official docs.